src/server/DBMetadata/mssql/msMetadataRetriever.js

```diff
@@ -63,8 +63,7 @@ async function getDbPool (connString) {
 
 	if (poolCache[digest]) return poolCache[digest];
 
-	const pool = await new sql.ConnectionPool(connString).connect();
-	poolCache[digest] = pool;
+	const pool = await new sql.ConnectionPool(connString).connect().then(poolCache[digest] = pool);
```

Comment:
This is incorrect, there's no reason to use .then, it's an async function that uses await. The original version is correct.